### ISSUE TITLE
Custom Kanban

### ISSUE THREAD

**Issue by Rigo-m** [volta-net/feedback#23]
One should be able to customise the kanban statuses

**Comment by benjamincanac** [comment]
Hello @Rigo-m,

Due to the nature of Volta and all of the automation included in the kanban board, the statuses are not customizable.

However, you can easily choose to hide specific statuses from the `View` select:
![CleanShot 2022-11-16 at 11.34.19@2x.png](https://volta.s3.fr-par.scw.cloud/Clean_Shot_2022_11_16_at_11_34_19_2x_03607a89a8.png)

**Comment by Rigo-m** [comment]
I see.
As an afterthought, would renaming them be possible somehow? It would make room for a bit of customisation of the workflow at least "UI side"

**Comment by benjamincanac** [comment]
This should be possible yes, it might not be a top priority right now but I'll keep this in mind. 

It could be packed with the feature mentioned in #24.

**Comment by teziovsky** [comment]
Agree! Not everyone needs a lot of statuses, it could be customizable 🚀

**Comment by foxted** [comment]
Could it pull the list of existing statuses from GitHub? Just tried to plug in an existing board, but since the statuses aren't in sync, had to backtrack :/
